spec/app/domain/authentication/authn-azure/application_identity_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Authentication::AuthnAzure::ApplicationIdentity do
+  let(:subscription_id_annotation) { double("SubscriptionIdAnnotation") }
+  let(:subscription_id_annotation_empty) {double("SubscriptionIdAnnotationEmpty") }
+  let(:subscription_id_service_id_scoped_annotation) { double("SubscriptionIdServiceIdAnnotation") }
+
+  let(:good_service_id) { "MockService" }
```

Comment:
as long as you don't have a bad service id this name doesn't make sense. Change to `test_service_id`